tests/installation/partitioning_raid.pm

```diff
@@ -162,31 +166,38 @@ sub run() {
         assert_screen 'partitioning-size';
         send_key 'ctrl-a';
```

Comment:
ok, here I see a potential synchronisation issue. Because there is a key press that should have an effect on the selected field I strongly suggest to guard that send_key before trying to type further. It is not code that you touched already but IMHO it correlates with what you want to change and fix here anyway.